ouroboros-consensus/src/Ouroboros/Consensus/Util/IOLike.hs

```diff
@@ -1,13 +1,88 @@
+{-# LANGUAGE FlexibleContexts #-}
+
 module Ouroboros.Consensus.Util.IOLike (
-    module X
```

Comment:
Any reason for not simply re-export the modules, except for documentation purposes?

Reply:
Because we don't want to export `MonadAsync` and co (that should be `IOLike`). Also, it's nice to see I think at a glance the API we actually depend on.